libraries/datahandler/src/main/resources/sw360.properties

```diff
@@ -53,3 +53,6 @@
 ## This property is used to enable the dependency management function.
 ## To enable the function, you should also run the migration script (scripts/migrations/060_migrate_project_dependency_network.py).
 #enable.flexible.project.release.relationship=true
+
+## This property is used to enable or disable report should receive in mail or direct download.
+send.project.spreadsheet.export.to.mail.enabled=true
```

Comment:
why adding this field again as this is already added in backend's sw360.properties file?

Reply:
@rudra-superrr @keerthi-bl , this field will be required when the rest server is separated?

Reply:
No @GMishx , its not required. It is added in different file path, there its enough.

Reply:
In that case it can be deleted from here

Reply:
Deleted and resolved already!

Reply:
Umm... I can still see the changes in PR

![image](https://github.com/eclipse-sw360/sw360/assets/18077542/c0b10fca-5bfd-45a3-b2c0-c61dc65bc4c1)

Maybe you forgot to add the changes to commit?

Reply:
Oh, the properties are expected there. The comment is on libraries.

Sorry for the confusion.